debug-head.html

```diff
@@ -1,5 +1,11 @@
         <link type="text/css" rel="stylesheet" href="./css/bootstrap.css">
-        <link type="text/css" rel="stylesheet" href="./css/style.css">
+        <link type="text/css" rel="stylesheet" href="./css/analysis.css">
+        <link type="text/css" rel="stylesheet" href="./css/footer.css">
+        <link type="text/css" rel="stylesheet" href="./css/general.css">
+        <link type="text/css" rel="stylesheet" href="./css/navbar.css">
+        <link type="text/css" rel="stylesheet" href="./css/nojs.css">
+        <link type="text/css" rel="stylesheet" href="./css/rtl.css">
+        <link type="text/css" rel="stylesheet" href="./css/translation.css">
```

Comment:
Could you order these a bit more? general, navbar, footer, rtl, translation, analysis makes sense to me.